k9mail-library/src/main/java/com/fsck/k9/mail/Message.java

```diff
@@ -92,6 +92,8 @@ public void setInternalDate(Date internalDate) {
 
     public abstract Address[] getRecipients(RecipientType type) throws MessagingException;
 
+    public abstract Address[] getListPost();
```

Comment:
Please don't add more utility methods to `Message`. Instead you could create a method `ListHeaders.getListPostAddress(Message)` that gets the appropriate header from the message.